entity-management-web/src/test/java/eu/europeana/entitymanagement/mongo/EntityRecordRepositoryTest.java

```diff
@@ -35,7 +35,7 @@ public void insertEntityToMongoDB() {
 		EntityRecordRepository entityRecordRepository = context.getBean(EntityRecordRepository.class);
```

Comment:
please use the @Resource(name=AppConfig.BEAN...) instead of context.getBean

Reply:
This class has been deleted.